projects/storefrontapp-e2e-cypress/package.json

```diff
@@ -13,11 +13,11 @@
+    "cy:run:ci": "cypress run --config-file cypress.ci.json --record --key $CYPRESS_KEY --tag \"b2c,all\" --parallel --ci-build-id $BUILD_NUMBER --group B2C --spec \"cypress/integration/!(vendor|b2b|ssr)/**/*e2e-spec.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-b2c-[hash].xml",
     "cy:run:ci:core": "cypress run --config-file cypress.ci.json --record --key $CYPRESS_KEY --tag \"b2c,core\" --parallel --group B2C --spec \"cypress/integration/!(vendor|b2b|ssr)/**/*.core-e2e-spec.ts\"",
-    "cy:run:ci:b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,all\" --parallel --group B2B --spec \"cypress/integration/b2b/regression/**/*e2e-spec.ts\"",
+    "cy:run:ci:b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,all\" --parallel --ci-build-id $BUILD_NUMBER --group B2B --spec \"cypress/integration/b2b/regression/**/*e2e-spec.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-b2b-[hash].xml",
     "cy:run:ci:core:b2b": "cypress run --config-file cypress.ci.json --env BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,core\" --parallel --group B2B --spec \"cypress/integration/b2b/regression/**/*.core-e2e-spec.ts\"",
-    "cy:run:ci:ssr": "cypress run --config-file cypress.ci.json --config baseUrl=http://localhost:4000 --record --key $CYPRESS_KEY --tag \"ssr,all\" --parallel --group SSR --spec \"cypress/integration/ssr/pages.core-e2e-spec.ts\"",
+    "cy:run:ci:ssr": "cypress run --config-file cypress.ci.json --config baseUrl=http://localhost:4000 --record --key $CYPRESS_KEY --tag \"ssr,all\" --parallel --ci-build-id $BUILD_NUMBER --group SSR --spec \"cypress/integration/ssr/pages.core-e2e-spec.ts\" --reporter junit --reporter-options mochaFile=results/e2e-test-ssr-[hash].xml",
     "cy:run:ci:ccv2": "cypress run --config-file cypress.ci.json --reporter junit --reporter-options mochaFile=results/test-results-[hash].xml --env API_URL=https://api.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud --record --key $CYPRESS_KEY --tag \"b2c,ccv2\" --parallel --group CCV2-B2C --spec \"cypress/integration/cx_ccv2/regression/b2c/**/*e2e-spec.ts\"",
     "cy:run:ci:ccv2-b2b": "cypress run --config-file cypress.ci.json --reporter junit --reporter-options mochaFile=results/test-results-[hash].xml --env API_URL=https://api.cg79x9wuu9-eccommerc1-p1-public.model-t.myhybris.cloud,BASE_SITE=powertools-spa,OCC_PREFIX_USER_ENDPOINT=orgUsers --record --key $CYPRESS_KEY --tag \"b2b,ccv2\" --parallel --group CCV2-B2B --spec \"cypress/integration/cx_ccv2/regression/b2b/**/*e2e-spec.ts\"",
     "cy:run:ci:cds": "cypress run --config-file cypress.ci.json --record --key $CYPRESS_KEY --tag \"2011,b2c,all-cds\" --group CDS --spec \"cypress/integration/vendor/cds/**/*.core-e2e-spec.ts\"",
```

Comment:
The 3 XML files (e2e-test-b2c-[hash].xml, e2e-test-b2b-[hash].xml, e2e-test-ssr-[hash].xml) don't cover vendor folder.
 Should we also generate a e2e-test-.xml for tests within integration/vendor folder as well like this vendor/cds test?

Reply:
to be honest, I would say no since we don't necessarily own that package. But thinking about it, it is still Spartacus right?

All these vendors have different setup to get them working on a e2e test as we don't have a source of truth for all features that includes 'vendor' packages.

So my answer is no. But we can follow this up when Gil gets back